clouddriver-oracle/src/main/groovy/com/netflix/spinnaker/clouddriver/oracle/deploy/converter/DeleteOracleLoadBalancerAtomicOperationConverter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
Can we ensure that these license headers follow the same convention as the rest of our other source.

```
/*
 * Copyright 2018 xxx
 *
 * Licensed under the Apache License, Version 2.0 (the "License")
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
@markxnelson

Reply:
My bad ... this stood out as being similar but different from the standard apache license header. I knew there was some requirement to have oracle in the header but gapped on it being this different.

Reply:
no problem :)